runner/docker/Dockerfile.live-base-scope

```diff
@@ -20,32 +20,14 @@ RUN apt update && apt install -yqq --no-install-recommends wget curl ca-certific
     apt purge -y --auto-remove wget curl && \
     apt clean && rm -rf /var/lib/apt/lists/*
 
-# Install Node.js for frontend build
-RUN apt update && apt install -yqq --no-install-recommends curl ca-certificates gnupg && \
-    curl -fsSL https://deb.nodesource.com/setup_20.x | bash - && \
-    apt install -yqq nodejs && \
-    apt clean && rm -rf /var/lib/apt/lists/*
-
-RUN conda run -n comfystream pip install --no-cache-dir diffusers==0.35.0 transformers==4.56.0
-
-# Install Scope dependencies into the comfystream environment
-# Clone scope repo and install with uv
-# Note: --active flag ensures uv installs packages into the active conda environment instead of
-# creating a separate .venv, making packages available alongside pip-installed packages
+# Install Scope lib into the comfystream environment
 ARG SCOPE_VERSION=main
-RUN git clone --depth 1 --branch ${SCOPE_VERSION} https://github.com/daydreamlive/scope.git /workspace/scope && \
-    cd /workspace/scope && \
-    conda run -n comfystream uv sync --frozen --active && \
-    cd frontend && \
-    npm install && \
-    npm run build && \
-    cd /workspace && \
-    rm -rf /workspace/scope/.git
+RUN conda run -n comfystream uv pip install --no-cache git+https://github.com/daydreamlive/scope.git@${SCOPE_VERSION}
```

Comment:
FTR I'm simplifying this further on https://github.com/livepeer/ai-runner/pull/854 but for now we need this conda uv pip frankenstein